### PR TITLE
Opics configurations for eval 5

### PR DESCRIPTION
**Pull request by jaudick**

### CODE REVIEW

File: mako/variables/default.yaml [hunk 1]
```diff
@@ -1,21 +1,18 @@
 workers: 8
 region: us-east-1
-clusterUser: ""
+clusterUser: "jaud"
```
Comment (ThomasSchellenbergNextCentury) [inline mako/variables/default.yaml:3]: I think this needs to be reset before we merge this branch.

Comment (ThomasSchellenbergNextCentury) [reply mako/variables/default.yaml:3]: Also can't you use the `-u` command line argument for this? Or am I mistaken?

Comment (kdrumm) [reply mako/variables/default.yaml:3]: I noticed this and thought i'd chime in.  There is a gitignored file that is also read for configuration and the `clusterUser` is typically put in there so it stays only for the users machine.  I forget what the file name is, but i know its hidden in the docs somewhere (probably user.yaml or something like that).  @ThomasSchellenbergNextCentury 's idea is also sound if one prefers that method.

**Comment by jaudick** [comment]
This should be ready to merge now

File: mako/eval.yaml [hunk 2]
```diff
@@ -1,7 +1,5 @@
 base:
   varset: ['opics']
 eval-groups:
-  - metadata: ['level1', 'level2']
-    parent-dir: 'mako-test/parent'
-  - metadata: ['level2', 'oracle']
-    dirs: ['mako-test/dirs/dir1', 'mako-test/dirs/dir2']
+  - metadata: ['level2']
+    dirs: ['']
```
Comment (rartiss55) [inline mako/eval.yaml:5]: I think we should keep the old version to illustrate how different metadata levels/directory properties work if needed.

**Pull request merged**